src/main/webapp/populateData.js

```diff
@@ -90,12 +90,24 @@ function populateGmail() {
  * Populate Tasks container with user information
  */
 function populateTasks() {
-  // Get Container for Tasks content
-  const tasksContainer = document.querySelector('#tasks');
+  let fetchFrom;
+  const select = document.querySelector('#tasks-select');
+  // Cast from HTMLOptionsCollection to Array
+  const options = Array(...select.options);
 
-  // Get list of tasks from user's Tasks account
-  // and display the task titles from all task lists on the screen
-  fetch('/tasks')
+  if (options.length === 0) {
+    fetchFrom = '/tasks';
+  } else {
+    const selectedOptions = [];
+    options.forEach((option) => {
+      if (option.selected) {
+        selectedOptions.push(option.value);
+      }
+    });
+    fetchFrom = '/tasks?selected=' + selectedOptions.join();
```

Comment:
Nit: "selected" is a very generic parameter name. I suggest something more specific like `taskListsSelected` or just `taskLists`.

Reply:
Agreed, thanks!

Reply:
`RESOLVED`